source/tyr/tyr/resources.py

```diff
@@ -2053,66 +2053,111 @@ def delete(self, id=None):
 
 
 class StreetNetworkBackend(flask_restful.Resource):
-    @marshal_with(streetnetwork_backend_list_fields)
-    def get(self, id=None):
-        if id:
-            try:
-                return {'streetnetwork_backends': [models.StreetNetworkBackend.find_by_id(id)]}
-            except sqlalchemy.orm.exc.NoResultFound:
-                return {'streetnetwork_backends': []}, 404
+    def _get_user_by_id(self, backend_id):
+        return models.StreetNetworkBackend.query.get_or_404(backend_id)
```

Comment:
I don't understand why you use "_get_user_by_id" here. 
Isn't it find_by_id or get_by_id ?

Reply:
It should be _get_backend_by_id. Nice catch!